src/features/react-virtualized-renderers/fileNameCellRenderer.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { FormattedMessage } from 'react-intl';
+import { getFileExtension } from '../../utils/file';
+import commonMessages from '../../common/messages';
+import FileIcon from '../../icons/file-icon';
+import Link from '../../components/link/Link';
+
+import baseCellRenderer from './baseCellRenderer';
+
+import type { FileNameCellRendererCellData, FileNameCellRendererParams } from './flowTypes';
+
+import './FileNameCell.scss';
+
+const fileNameCellRenderer = (cellRendererParams: FileNameCellRendererParams) =>
+    baseCellRenderer(cellRendererParams, (cellValue: FileNameCellRendererCellData) => {
+        const { id, name, isExternal } = cellValue;
+        const extension = getFileExtension(name);
+        const href = id ? `/file/${id}` : '';
```

Comment:
Should we render the anchor at all if the `href` value is empty? Should it be a `p` tag or something, instead?